### PR TITLE
My enumerables

### PR DESCRIPTION
**Pull request by brytebee**
I have implemented the corrections as the last review instructs in the previous pull request.

As I tried renaming the branches as suggested, the initial pull request below was closed.

https://github.com/brytebee/the_enumerables/pull/1

This is the new one.

### CODE REVIEW

File: enumerables.rb [hunk 1]
```diff
@@ -0,0 +1,268 @@
+# rubocop: disable all metrics
+
```
Comment (mimipeshy) [inline enumerables.rb:2]: Remove this line

File: enumerables.rb [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+} # should return 604800
+longest = %w{ cat sheep bear }.inject do |memo, word|
+  memo.length > word.length ? memo : word
+end
+puts longest
+(5..10).inject(1) { |product, n| 
+p product * n 
+} #=> 151200
+
+# rubocop: enable all metrics
```
Comment (mimipeshy) [inline enumerables.rb:268]: remove line 268 as well, work on all errors that the linter raises

**Comment by mimipeshy** [comment]
### STATUS: CHANGES REQUESTED 🔴 ♻️ 
Hello, there @brytebee 👋🏾 👋🏾 a great job so far on this milestone.
✔️ Great job in explaining why you had to close the previous PR, that's professional of you
✔️ Great job on working on all changes requested, your methods work well

However, one more change is needed before marking the project as complete

- [ ] You are missing` #multiply_els` which accepts an array as an argument and multiplies all the elements of the array together using #my_inject. It's throwing an undefined methodError. Kindly add it.
- [ ] You are not supposed to disable rubocop, you are supposed to work on the errors it raises. This can be easily fixed by `rubocop -a --fix`.If you must disable it, it has to be a specific metric and the section of the codebase it affects. Not from top to bottom of the file.
RuboCop cops are meant to catch bad code and enforce best practices and the rubocop.yml file provided by [Microverse linters repo](https://github.com/microverseinc/linters-config/blob/master/ruby/.rubocop.yml) doesn't enforce strict rules.


That's all :smile: kindly submit another review when done.

_Feel free to reach out to me on slack if you have any questions or something is unclear, or just comment on this PR and tag my name and I will be glad to help_
Regards
Peris Ndanu

File: enumerables.rb (hunk 1 above)
Comment (mimipeshy) [inline enumerables.rb]: remove this line, fix all rubocop errors

File: enumerables.rb (hunk 2 above)
Comment (mimipeshy) [inline enumerables.rb]: remove line 268 as well, work on rubocop errors that arise

**Review by umairarshadbutt — APPROVED** [review]
# **Status: PROJECT APPROVED** :champagne:
Excellent job on this project guys. :fireworks: Keep it up.
##
**All the required tasks have been fulfilled in this project. I will go ahead and mark this as complete.**
- Congratulations on the splendid job @brytebee ! :+1: 
- You can now merge to master and submit the project completion form :rocket: . I wish you all the very best in your next tasks :1st_place_medal:
##
**If you have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Pull request merged**